mappings/net/minecraft/entity/damage/DamageSource.mapping

```diff
@@ -72,3 +72,4 @@ CLASS none/rp net/minecraft/entity/damage/DamageSource
 	METHOD r isScaledWithDifficulty ()Z
 	METHOD s setMagic ()Z
 	METHOD t setMagic ()Lnone/rp;
+	METHOD u isByCreativePlayer ()Z
```

Comment:
I wasn't sure about this either. I didn't want to name it `byCreativePlayer` because there's the very similarly named static helper functions like `byMob`.